vochain/appsetup.go

```diff
@@ -61,7 +66,9 @@ func (app *BaseApplication) SetDefaultMethods() {
 	})
 	app.SetFnMempoolPrune(app.fnMempoolRemoveTxTendermint)
 	app.SetFnSendTx(func(tx []byte) (*cometcoretypes.ResultBroadcastTx, error) {
-		result, err := app.NodeClient.BroadcastTxSync(context.Background(), tx)
+		ctx, cancel := context.WithTimeout(context.Background(), transactionSendTimeout)
+		defer cancel()
+		result, err := app.NodeClient.BroadcastTxSync(ctx, tx)
```

Comment:
why is this change included in this commit? please split the changes of `vochain/appsetup.go` into a different commit, or if it's needed somehow for the keykeeper refactor, then add a comment maybe to transactionSendTimeout const? or at least to the commit message

Reply:
I have been stock on a deadlock for a while... Then I realized we do not have context on sendTx, which might be an issue. So I just added it.